src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -4340,81 +4340,116 @@ module Queries = struct
           | Some _, Some _ ->
               Deferred.Result.fail "Cannot specify both state hash and height"
         in
-        let block = Transition_frontier.Breadcrumb.block breadcrumb in
-        let blockchain_length = Mina_block.blockchain_length block in
-        let global_slot =
-          Mina_block.consensus_state block
-          |> Consensus.Data.Consensus_state.curr_global_slot
+        let target_block =
```

Comment:
I renamed a bunch of variables so that it's clear that we refer to "target" vs. best tip